updateJsonSchema.js

```diff
@@ -0,0 +1,38 @@
+const offer_contents = (await OfferContent.find())
+
+async function updatePolls () {
+	for (let i = 0; i < offer_contents.length; i++) {
+		const oc = offer_contents[i];
+		const pageTypeNum = 0;
+		if (oc.contentType !== 'poll') { continue; }; // I divided polls and quizzes to two separate calls
+		const content = JSON.parse(oc.content);
+		content.pages.forEach(page => {
+			if (page.type !== pageTypeNum) { return false; }
+			page.rewardText = page.title.replace(/ \${amount}.*/, '');
```

Comment:
If this is the only difference between quiz and poll I would just do an if statement here instead of code duplication.

Reply:
I ran the code separately for more caution reasons, polls and after them quizzes. 
I can execute the function with parameters: `pageTypeNum` and `contentType`, but it seemed to much ifs